docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  postgres:
+    image: mysql:latest
+    ports: 
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: zintranet
+      MYSQL_ROOT_PASSWORD: root 
```

Comment:
We could use env vars here (docker-compose reads `.env`), that would ensure that the db container and the app are always in sync for local dev.